test/infrastructure/vcsim/controllers/envvar_controller.go

```diff
@@ -287,14 +293,74 @@ func clusterEnvVarSpecGovmomiVariables(c *vcsimv1.ClusterEnvVarSpec) map[string]
 	}
 
 	// NOTE: omitting cluster Name intentionally because E2E tests provide this value in other ways
-	vars["VSPHERE_DATACENTER"] = vcsimhelpers.DatacenterName(datacenter)
+	// Vars below can be named only so they should be used as is
```

Comment:
named -> names?
as is -> as they are
potentially reword this sentence for better clarity

Reply:
reworded to `	// The var VSPHERE_COMPUTE_CLUSTER cannot be used as moid during the tests.`